packages/react-server/core/FramebackController.js

```diff
@@ -228,40 +229,19 @@ class FramebackController extends EventEmitter {
 
 	createFrame(url){
 
-		this.frame = document.createElement("iframe");
+		const frame = this.frame = document.createElement("iframe");
```

Comment:
It looks like the main changes here are some 1:1 refactorings, plus removing the `this.frame.contentWindow` unload listener layer of indirection?